2_gitlab/cloudbuild-destroy.yaml

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 3600s # 1-hr
+tags:
+  - modern-cicd-anthos
+  - modern-cicd-gitlab
```

Comment:
for these secondary tags I would leave them as `gitlab`. The filtering based on these tags would then be simpler:

```
gcloud builds list --filter "tags='modern-cicd-anthos' AND 'gitlab'"
```

Reply:
Updates coming to remove "modern-cicd" prefix to gitlab